src/Apps/CollectorProfile/Routes/Saves2/Components/CreateNewListModal/AddArtworksModal.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react"
+import { Flex, Text, ModalDialog, Spacer, Button } from "@artsy/palette"
+import { SortFilter } from "Components/SortFilter"
+import { ArtworksList, TEMP_ARTWORKS } from "./ArtworksList"
+import { useTranslation } from "react-i18next"
+
+interface AddArtworksModalProps {
+  visible: boolean
+  onClose: () => void
+  onComplete: () => void
+  listName: string
+}
+
+const AddArtworksModal: React.FC<AddArtworksModalProps> = ({
+  listName,
+  onClose,
+}) => {
+  const { t } = useTranslation()
+
+  const handleSave = () => {
+    onClose()
+  }
+
+  return (
+    <ModalDialog
+      m={["0", "auto"]}
+      dialogProps={{
+        width: ["100%", 700],
+        height: ["100%", "auto"],
+        maxHeight: [null, 800],
+      }}
+      onClose={onClose}
+      title={t("collectorSaves.addArtworksModal.modalTitle", {
```

Comment:
I still couldn't find a way to split title in two lines. @dimatretyak I am thinking to allow passing a component instead of a string as a title, like header and footer do. What do you think about it?

Reply:
unfortunately, we will not be able split title in two lines with current implementation (we need to add changes to the `palette`). Maybe it's better to discuss this point with Ryan and use only one line (e.g. just `Add saved works to this list`)?